src/pages/scout/PreMatch.tsx

```diff
@@ -68,9 +68,20 @@ const PreMatch = () => {
                     {context.meta.teamNumber}
                 </span>
                 <span> in match </span>
-                <span>
-                    {context.meta.matchId}
-                </span>
+                <Select
```

Comment:
You might want to consider using a standard variant of the select menu, as well as enclosing it with a FormControl component. When you do that make sure the labelIds match up.
Example: https://mui.com/material-ui/react-select/#system-SelectVariants.tsx